index.js

```diff
@@ -9,10 +10,26 @@ function CoffeeScriptFilter (inputTree, options) {
   Filter.call(this, inputTree, options)
   options = options || {}
   this.bare = options.bare
+  this.options = options;
 }
 
 CoffeeScriptFilter.prototype.extensions = ['coffee', 'litcoffee', 'coffee.md']
 CoffeeScriptFilter.prototype.targetExtension = 'js'
+CoffeeScriptFilter.prototype.baseDir = function() {
+  return __dirname;
```

Comment:
Ah. I agree it seems reasonable to make it on-by-default. (It'd seem very strange for the user to enable or disable persistent caching on a per-plugin basis.) Is that what you do with other plugins?

> the tests should stress it.

Yes, definitely, at least a smoke test to exercise the code path.